analytics.go

```diff
@@ -143,7 +142,7 @@ func New(key string) *Client {
 		now:      time.Now,
 		uid:      uid,
 	}
-
+	go c.loop()
```

Comment:
It's also weirded me out that any of our libs would have an interval. It feels like it makes more sense to just limit the number of outstanding requests (since that is the desired bottleneck) and then take messages off up to `max_batch` at a time. 

Thoughts? That feels the most natural to me anyway, just having a consumer constantly running, and taking messages off in batches of up to 100.